__DEFINES/global.dm

```diff
@@ -403,6 +403,9 @@ var/station_does_not_tip = FALSE
 var/malf_radio_blackout = FALSE
 var/malf_rcd_disable = FALSE
 
+//changeling hivemind bank
+var/list/hivemind_bank = list()
```

Comment:
Shouldn't this be a var on the hivemind faction?

Reply:
definitely

Reply:
well?